integration-requirements.txt

```diff
@@ -2,7 +2,8 @@
 behave
 jsonschema
 PyHamcrest
-pycloudlib @ git+https://github.com/canonical/pycloudlib.git@756a2c2de044ca60eaa7cdc76653d23a1339dc0a
+pycloudlib @ git+https://github.com/canonical/pycloudlib.git@d8c7654dbef1a7daf47417337e50515e6842aed6
```

Comment:
This needs to be minimally 44206bb95c49901d994c9eb772eba07f2a1b6661 to get Azure Jammy image support